dockerfiles/Dockerfile_alpine

```diff
@@ -1,8 +0,0 @@
-FROM --platform=i386 i386/alpine
-RUN apk update && apk upgrade
-RUN apk add linux-virt
-RUN apk add openrc
```

Comment:
I see the file was committed by mistake, just remove it from the previous commit of the PR instead